examples/playground/components/json-highlighter.tsx

```
@@ -0,0 +1,31 @@
+          className = 'text-blue-500'; // boolean - using blue-500 (#3b82f6)
+        } else if (/null/.test(match)) {
+          className = 'text-red-500'; // null - using red-500 (#ef4444)
+        }
+        return `<span class="${className}">${match}</span>`;
+      },
+    );
+
+  return (
+    <pre className="bg-gray-500/5 rounded-md p-4 text-sm">
+      <code
+        dangerouslySetInnerHTML={{ __html: jsonString }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)